src/test/spec/response.ts

```diff
@@ -19,29 +19,94 @@ describe('response.ts', () => {
     expect(Response.transformTags('Hey <*>')).to.equal('^Hey ([\\s\\S]*?)$');
     expect(Response.transformTags('Hey <WORD> there')).to.equal('^Hey ([^ ]+?) there$');
   });
-  it('should instantiate <CARDS> correctly', () => {
-    expect(new Response(' <CARDS> ').responseType).to.equal(MessageType.Card);
-  });
-  it('should instantiate <IMAGE> correctly', () => {
-    expect(new Response(' <IMAGE> ').responseType).to.equal(MessageType.Image);
-  });
-  it('should match <CARDS> correctly', () => {
-    expect(new Response(' <CARDS> ').responseType).to.equal(MessageType.Card);
-  });
   it('should match <IMAGE> correctly', () => {
     expect(new Response(' <IMAGE> ').matches({
       user: null,
       text: null,
-      messageTypes: [MessageType.Image],
+      attachments: [Attachment.Image],
     })).to.be.true;
   });
   it('should match <CARDS> correctly', () => {
     expect(new Response(' <CARDS> ').matches({
       user: null,
       text: null,
-      messageTypes: [MessageType.Card],
+      attachments: [Attachment.Cards],
     })).to.be.true;
   });
+  it('should ignore text with <IMAGE> if no text is expected', () => {
```

Comment:
Actually I think this should be a breaking change, and we can increment the major version of autobot to suggest this, and put a note in README.md